### PR TITLE
Implement missing parameters of commands-related methods

### PR DESCRIPTION
**Pull request by zry98**
Implement the missing two optional parameters (`scope` and `language_code`) of methods `getMyCommands`, `setMyCommands` and `deleteMyCommands` according to the official document: https://core.telegram.org/bots/api#setmycommands.

### CODE REVIEW

File: options.go [hunk 1]
```diff
@@ -324,3 +324,28 @@ func (b Btn) Reply() *ReplyButton {
...
+}
+
+// BotCommandScope object represents a scope to which bot commands are applied.
+type BotCommandScope struct {
+	Type   string `json:"type"`
+	ChatID int64  `json:"chat_id,omitempty"`
+	UserID int64  `json:"user_id,omitempty"`
+}
+
+const (
```
Comment (demget) [inline options.go:338]: Remove `Bot` prefix from constants and all command-related types.

File: options.go [hunk 2]
```diff
@@ -324,3 +324,28 @@ func (b Btn) Reply() *ReplyButton {
...
+	// BotCommandScope types
```
Comment (demget) [inline options.go]: Move a comment before the `const` declaration.

File: options.go [hunk 3]
```diff
@@ -324,3 +324,28 @@ func (b Btn) Reply() *ReplyButton {
...
+	BotCommandScopeDefault               = "default"
+	BotCommandScopeAllPrivateChats       = "all_private_chats"
+	BotCommandScopeAllGroupChats         = "all_group_chats"
+	BotCommandScopeAllChatAdministrators = "all_chat_administrators"
+	BotCommandScopeChat                  = "chat"
+	BotCommandScopeChatAdministrators    = "chat_administrators"
+	BotScopeChatMember                   = "chat_member"
```
Comment (demget) [inline options.go]: It's `CommandScopeChatMember`.

File: options.go [hunk 4]
```diff
@@ -324,3 +324,28 @@ func (b Btn) Reply() *ReplyButton {
...
+}
+
+const (
+	// BotCommandScope types
+	BotCommandScopeDefault               = "default"
+	BotCommandScopeAllPrivateChats       = "all_private_chats"
+	BotCommandScopeAllGroupChats         = "all_group_chats"
+	BotCommandScopeAllChatAdministrators = "all_chat_administrators"
+	BotCommandScopeChat                  = "chat"
+	BotCommandScopeChatAdministrators    = "chat_administrators"
```
Comment (demget) [inline options.go]: Rename to `CommandScopeChatAdmin`.

File: options.go [hunk 5]
```diff
@@ -324,3 +324,28 @@ func (b Btn) Reply() *ReplyButton {
...
+	ChatID int64  `json:"chat_id,omitempty"`
+	UserID int64  `json:"user_id,omitempty"`
+}
+
+const (
+	// BotCommandScope types
+	BotCommandScopeDefault               = "default"
+	BotCommandScopeAllPrivateChats       = "all_private_chats"
+	BotCommandScopeAllGroupChats         = "all_group_chats"
+	BotCommandScopeAllChatAdministrators = "all_chat_administrators"
```
Comment (demget) [inline options.go]: Rename to `CommandScopeAllChatAdmin`.

**Review by demget — CHANGES_REQUESTED** [review]
This is a significant help, thank you. I left some notes, please fix them and we're happy to merge.

**Comment by zry98** [comment]
OK done, but is there any refactoring plan on method parameters? since there has been a lot of changes in the API, like these command related.

**Comment by demget** [comment]
Thank you. 

> is there any refactoring plan on method parameters?

What's about them? I see we now fully satisfy Bot API 5.3 changes.

**Review by demget — APPROVED** [review]

**Pull request merged**